src/controllers/CategoryFacetQueryController.ts

```diff
@@ -6,6 +6,7 @@ import { AllowedValuesPatternType } from '../rest/AllowedValuesPatternType';
 import { IGroupByValue } from '../rest/GroupByValue';
 import { Utils } from '../utils/Utils';
 
+// TODO: remove this file and any reference to it
```

Comment:
Is this TODO still needed?
Same for line 8 and 19 in `DynamicCategoryFacetQueryController.ts`

Reply:
Yes!